crates/steel-parser/src/tokens.rs

```diff
@@ -451,15 +447,15 @@ impl<T: Display> fmt::Display for TokenType<T> {
 }
 
 #[derive(Debug, Clone, PartialEq)]
-pub struct Token<'a, T> {
-    pub ty: TokenType<T>,
+pub struct TokenLike<'a, TY> {
```

Comment:
:star: We use this struct to return "stuff that looks like a token (like a tokenization error).